.github/workflows/content.yml

```diff
@@ -0,0 +1,133 @@
+              id: cache
+              uses: actions/cache@v2
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: ./node_modules
+                  key: modules-${{ hashFiles('package-lock.json') }}
+            - name: Install Node.js dependencies
+              run: |
+                  npm ci
+                  npm test
+
+    release-production:
+        needs: lint
+        timeout-minutes: 30
+        runs-on: ubuntu-latest
+        environment: production
+        steps:
+            - name: Checkout 🛎️
+              uses: actions/checkout@v2.3.1
+
+            - name: Setup Node
+              uses: actions/setup-node@v2.1.2
+              with:
+                  node-version: '14.x'
+
+            - name: Set version env variable
+              run: echo "GATSBY_DERIV_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                  path: ~/.npm
+                  key: ${{ runner.os }}-node-${{ secrets.CACHE_VERSION }}-${{ hashFiles('**/package-lock.json') }}
+                  restore-keys: |
+                      ${{ runner.os }}-node-${{ secrets.CACHE_VERSION }}
+
+            - name: Cache Gatsby
+              id: gatsby-cache-build
+              uses: actions/cache@v2
+              with:
+                  path: |
+                      public
+                      .cache
+                  key: ${{ runner.os }}-gatsby-build-${{ secrets.CACHE_VERSION }}-${{ github.run_id }}
+                  restore-keys: |
+                      ${{ runner.os }}-gatsby-build-${{ secrets.CACHE_VERSION }}
+
+            - run: npm ci
+            - run: npm run format
+            - run: npm run test
+            - run: npm run build
+
+            - name: Remove Storybook
+              uses: JesseTG/rm@v1.0.2
+              with:
+                  path: public/storybook
+
+            - uses: olegtarasov/get-tag@v2.1
+              id: tagName
+              # For using same tag for staging and production we need to uncomment these two below lines:
+              # with:
+              #   tagRegex: "production(.*)"
+            - name: Building docker image 🐳
+              run: |
+                  git fetch --prune --unshallow --tags
+                  echo "GIT_TAG_NAME=`echo $(git describe --tags --abbrev=0)`" >> $GITHUB_ENV
+                  docker build -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GIT_TAG_NAME .
+
+            - name: Pushing Image to docker hub 🐳
+              run: |
+                  echo ${{ secrets.DOCKERHUB_PASSWORD }}| docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+                  docker push  ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest
+                  docker push  ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GIT_TAG_NAME
```

Comment:
we need to remove docker build and push steps to dockerhub steps since it will overwrite latest docker image. we do need to the tag values though.